_posts/2015-07-20-self-organizing-team-2-footballs.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+author: curiousagilist
+authorname: Bob Allen
+title: Self Organizing Team Part 2 - Footballs
+preview: 'Below is a handout that was given at the beginning of the day. A second handout was given just after lunch. Together they define an exercise designed to require teams to work out and demonstrate these skills'
+---
+Below is a handout that was given at the beginning of the day. A second handout was given just after lunch. Together they define an exercise designed to require teams to work out and demonstrate these skills:
+
+
+1. Self organization of teams when given full reign (and full responsibility).
+2. How to work with what you're given, even when extremely sparse.
+3. How to deliver _iteratively (MVP)_.
+4. How to keep parts of the solution decoupled from one another.
+
+&lt;Handout #1 - Morning session&gt;
```

Comment:
Here is a quick JS fiddle that uses the [Materialize CSS](http://materializecss.com) library
https://jsfiddle.net/aqaL6L0u/3/
The advantage to this library is that it can also be used to replace the (minified and "stolen") CSS we now use on the [training page](http://coderetreat.org/pages/training/) for cards.